spec.md

```diff
@@ -198,6 +198,9 @@ help intermediate gateways determine how to route the events.
   semantics behind the data encoded in the URI is event producer defined.
 * Constraints:
   * REQUIRED
+* Examples
+    * urn:event:from:cloudevents/spec/pull/123
+    * urn:event:from:myapi/resourse/123
```

Comment:
Instead of two `urn` examples, what do you think about having each for a different scheme?
* `https://github.com/cloudevents/spec/pull/123`
* `urn:event:from:myapi/resourse/123`
* `mailto:cncf-wg-serverless@lists.cncf.io`

Reply:
Good point @cneijenhuis, I will do like this.